self_test/generic/vector_matrix_test/vector_matrix_test.cc

```diff
@@ -28,67 +28,62 @@
 #include "generic.h"
 
 using namespace oomph;
+using namespace std;
 
 //===start_of_main======================================================
 /// Driver code: Testing VectorMatrix class
 //======================================================================
 int main(int argc, char* argv[])
 {
-  std::ostringstream out_stream;
-  out_stream << "OUTPUT";
-
-  std::ofstream out_file;
-  out_file.open(out_stream.str().c_str());
-
   VectorMatrix<int> vector_matrix_1(3,4,-1);
 
   // test 1: nrow(), should be 3
-  out_file << vector_matrix_1.nrow() << "\n";
+  cout << vector_matrix_1.nrow() << endl;
```

Comment:
Ooh, thanks for sorting this out! It can be a right pain in the behind when debugging if the code breaks and some output is left sitting in the buffer (because it hasn't been flushed to the command-line by `std::endl`). (Well, if like me you used to be too lazy to recompile with debugging symbols and use a debugger...)